.github/workflows/build.yaml

```diff
@@ -0,0 +1,29 @@
+name: Build
+
+on:
+  pull_request:
+    branches: [ main ]
+  push:
+    branches: [ main ]
+
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Setup repo
+        uses: actions/checkout@v2
+
+      - name: Shared Build Steps
+        uses: momentohq/standards-and-practices/github-actions/shared-build@gh-actions-v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
nit: i'd really like it if you could add the readme template generator in this action.

If you use the pattern we are using in `laravel-cache` repo, where project_type is set to `other` instead of `sdk`, it should be pretty lenient and won't do much other than generating the header and footer for you, with badges.

https://github.com/momentohq/laravel-cache/blob/7f9f6e4c3443fa71b7d7b054b5d627d2403b3602/.github/workflows/on-pull-request.yaml#L22

also fine to do this in follow-up PR.

Reply:
ahh ok cool TIL did not know about the `other` project_type was why removed. I will add back!

Reply:
Gonna do in fast follow pr actually